### PR TITLE
Implement icon-font buttons.

### PR DESCRIPTION
**Pull request by jlhughes**
Defaults to 'Material Icons' from Google.
http://google.github.io/material-design-icons/

### CODE REVIEW

**Comment by kgrgreer** [comment]
Can you submit a demo to go along with this?

File: src/foam/u2/ActionView.js [hunk 1]
```diff
@@ -20,7 +20,14 @@ foam.CLASS({
   name: 'ActionView',
   extends: 'foam.u2.Element',
 
-  documentation: 'A button View for triggering Actions.',
+  documentation: function() {/*
```
Comment (mdittmer) [inline src/foam/u2/ActionView.js]: Backtick string for new documentation?

File: src/com/google/foam/demos/heroes/CitationView.js [hunk 2]
```diff
@@ -83,7 +86,9 @@ foam.CLASS({
     {
       name: 'removeHero',
       label: 'X',
-      code: function() { this.heroDAO.remove(this.data); }
+      speechLabel: 'delete',
+      iconFontName: 'delete_forever',
+      code: function() { this.heroDAO.remove(this.data);}
```
Comment (mdittmer) [inline src/com/google/foam/demos/heroes/CitationView.js]: nit: space before `}`.

File: src/foam/u2/ActionView.js [hunk 3]
```diff
@@ -114,11 +121,10 @@ foam.CLASS({
       } else if (this.iconFontName) {
         this.nodeName = 'i';
         this.cssClass(this.action.name);
-        this.cssClass(this.myClass(this.iconFontClass));
         this.cssClass(this.iconFontClass); // required by font package
         this.style({'font-family': this.iconFontFamily});
         this.add(this.iconFontName);
-        this.setAttribute('title', this.action.name); // hover text
+        this.setAttribute('title', this.action.speechLabel); // hover text
```
Comment (kgrgreer) [inline src/foam/u2/ActionView.js]: This is okay for now, but we probably want to distinguish between tooltip and speechlabel in the future.

**Comment by jlhughes** [comment]
Is anyone waiting on something for this?

**Comment by kgrgreer** [comment]
Fix the merge conflict and I'll merge.

File: src/foam/u2/ActionView.js [hunk 4]
```diff
@@ -92,9 +118,16 @@ foam.CLASS({
...
         this.start('img').attr('src', this.icon).end();
-      }
-      if ( this.showLabel )
+      } else if (this.iconFontName) {
+        this.nodeName = 'i';
+        this.cssClass(this.action.name);
+        this.cssClass(this.iconFontClass); // required by font package
+        this.style({'font-family': this.iconFontFamily});
+        this.add(this.iconFontName);
+        this.setAttribute('title', this.action.toolTip); // hover text
```
Comment (kgrgreer) [inline src/foam/u2/ActionView.js]: Is the tooltip dependent on the iconFontName being set? Can't it work for the other cases also?

**Comment by jlhughes** [comment]
- Resolved recent merge conflict.
- Set tooltip on all ActionViews.

**Pull request merged**